.gitignore

```diff
@@ -1,2 +1,3 @@
 coverage/
 node_modules/
+example.js
```

Comment:
I think it's fine to include an example.js file in a repo! Could just be to petstore, or w/e